docs/usage/compact.rst.inc

```diff
@@ -66,11 +66,11 @@ Depending on the amount of segments that need compaction, it may take a while,
 so consider using the ``--progress`` option.
 
 A segment is compacted if the amount of saved space is above the percentage value
-given by the ``--threshold`` option. If ommitted, a threshold of 10% is used.
+given by the ``--threshold`` option. If omitted, a threshold of 10% is used.
```

Comment:
this change should be removed in **this** file - it will be regenerated from source at release time.

Reply:
i reverted this change in the autogenerated file.